docs/user-guide/transformer_expressions.md

```diff
@@ -535,3 +549,397 @@ variables:
+
+#### Output
+New Column that has boolean type and same dimension with original column
+
+#### Examples
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: bool_column
+  expression: yourTableName.Col("avg_order_1_day").IsIn([2000, 3000])
+```
+Output:
+| bool_column |
+|----------------|
+| true |
+| true |
+| false |
+
+### StdDev
+`StdDev` is a function to calculate standard deviation from column values. The output will be single value
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+
+Standard Transformer Config:
+```
+variables:
+- name: std_dev
+  expression: yourTableName.Col("avg_cancellation_rate_30_day").StdDev()
+```
+Output: 0.0068475461947247
+
+### Mean
+`Mean` is a function to calculate mean value from column values. The output will be single value
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: mean
+  expression: yourTableName.Col("avg_order_1_day").Mean()
+```
+Output: 3000
+
+### Median
+
+`Median` is a function to calculate median value from column values. The output will be single value
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: median
+  expression: yourTableName.Col("avg_order_1_day").Median()
+```
+Output: 3000
+
+### Max
+
+`Max` is a function to find max value from column values. The output will be single value
+
+#### Input
+No Input
+
+#### Output
+Single value with float type
+
+#### Examples
+
+Suppose users have table `yourTableName`
+
+| restaurant_id | avg_order_1_day | avg_cancellation_rate_30_day |
+|---------------|-----------------|------------------------------|
+| 1 | 2000 | 0.02 |
+| 2 | 3000 | 0.005 |
+| 3 | 4000 | 0.006 |
+
+Standard Transformer Config:
+```
+variables:
+- name: max
+  expression: yourTableName.Col("avg_order_1_day").Max()
+```
+Output: 4000
+
+### MaxStr
```

Comment:
Curious - what is the motivation for having this function? Is it not possible for users to typecast the value of `max` to string using other functions? (From the doc, it seems we don't have type conversion functions, but confirming...)

Reply:
I think this is to expose all the method that already provided by `gota` for series that maybe useful for user, cmiiw @ariefrahmansyah. Yes unfortunately we don't have type conversion functions

Reply:
We have implicit type conversion in some parts of the standard transformer. But I think it'll be useful to have an explicit expression as well.